### PR TITLE
Document how to reduce number of mappers to avoid DDOS

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 1]
```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
 a number of mappers or reducers will be used to cover the file if the input file is sufficiently large.
 As each mapper will have their own client feeding to the endpoint, each with multiple threads,
 make sure the cluster can take the load.
-Each client will throttle to not overload the endpoint.
-Using a high number of effective thread (tasks times threads per mapper),
-can easily saturate feeding endpoints.
+Each client will throttle to not overload the endpoint but clients does not know how many total client there are so one can easily over saturate
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html:451]: Each client will throttle to not overload the endpoint**,** but **the** clients do not know how many total client**s** there are, so one can easily over**-**saturate *the* feeding endpoints.

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 2]
```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
...
+Each client will throttle to not overload the endpoint but clients does not know how many total client there are so one can easily over saturate 
+feeding endpoints.
 </p><p>
 There are two primary factors controlling feed load: the number of mappers or
-reducers, and the number of concurrent connections per mapper/reducer.
-Control the number of splits and thus number of mappers to make based on the input file size:
-<pre>
-SET mapreduce.input.fileinputformat.split.minsize 128*1024*1024
+reducers, and the number of concurrent connections per mapper/reducer. For large jobs with lots of data one might consider 
+control the number of splits and thus number of mappers to make based on the input file size. This can be done by
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html:456]: For large jobs with lots of data, one might consider control*ling* the number of splits and thus number of mappers to make based on the input file size.

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 3]
```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
...
-<pre>
-SET mapreduce.input.fileinputformat.split.minsize 128*1024*1024
+reducers, and the number of concurrent connections per mapper/reducer. For large jobs with lots of data one might consider 
+control the number of splits and thus number of mappers to make based on the input file size. This can be done by 
+controlling <a href="https://pig.apache.org/docs/latest/perf.html#combine-files">combine-files</a> by e.g adding
+<pre
+pig -Dpig.maxCombinedSplitSize=53687091200 -f feed.pig
 </pre>
-This is for Yarn and Hadoop v2, and will split the file after approximately 128Mb.
+This is for Yarn and Hadoop v2, and will combine the input so that each map process about 50G of data. This reduces the number of Vespa client instances.
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html:461]: processes

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 4]
```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
...
+	
 See the Pig and Yarn documentation for more information.
 </p><p>
 To set the number of concurrent connections per mapper/reducer,
-set <a href="#vespa.feed.connections">vespa.feed.connections</a>.
+set <a href="#vespa.feed.connections">vespa.feed.connections</a>. The default is 1 and rarely needs to be tuned upward. 
 </p>
 
-One way to limit the number of parallel tasks feeding to Vespa for large job is to use <em>PARALLEL</em> directive of PIG
+Another way to limit the number of parallel tasks feeding to Vespa for large job is to use <em>PARALLEL</em> directive of PIG
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html:469]: largs jobs

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 5]
```diff
@@ -448,38 +448,32 @@ <h3 id="efficiency">Efficiency</h3>
...
 
 <pre>
-data = LOAD '/projects/comms_psearch/psearch_suggest/hiveDB/ext_us_users_content_score/*' USING PigStorage('\u0001') AS (guid:chararray, query:chararray, to:double);
+data = LOAD '/some/data/*' USING PigStorage('\u0001') AS (guid:chararray, text:chararray, score:double);
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html:472]: LOL that this wasn't changed previously

**Review by lesters — APPROVED** [review]
Great! Please consider comments.

**Comment by jobergum** [comment]
Thanks @lesters

**Pull request merged**